src/main/java/comparator/PageComparator.java

```diff
@@ -34,43 +45,74 @@
     }
 
 
-    public static void main(String[] args) throws IOException {
+    public static void main(String[] args) throws Exception {
         if (!isValidInput(args)) {
             printUsage();
             return;
         }
         String url1 = args[0];
         String url2 = args[1];
-        String filenamePrefix = String.valueOf(Instant.now().toEpochMilli());
 
         WebDriverManager.chromedriver().setup();
         WebDriver driver = new ChromeDriver();
         driver.get(url1);
         driver.manage().window().maximize();
 
         Screenshot screenshot1 = new AShot().shootingStrategy(shootingStrategy).takeScreenshot(driver);
-        File file1 = new File(filenamePrefix + FIRST_PAGE + FILE_NAME_EXTENSION);
+        File file1 = new File(FILE_NAME_PREFIX + FIRST_PAGE + FILE_NAME_EXTENSION);
         ImageIO.write(screenshot1.getImage(), PNG_FORMAT, file1);
 
         driver.get(url2);
         Screenshot screenshot2 = new AShot().shootingStrategy(shootingStrategy).takeScreenshot(driver);
-        File file2 = new File(filenamePrefix + SECOND_PAGE + FILE_NAME_EXTENSION);
+        File file2 = new File(FILE_NAME_PREFIX + SECOND_PAGE + FILE_NAME_EXTENSION);
         ImageIO.write(screenshot2.getImage(), PNG_FORMAT, file2);
 
 
         ImageDiff diff = new ImageDiffer().makeDiff(screenshot1, screenshot2);
         BufferedImage diffImage = diff.getMarkedImage();
-        File result = new File(filenamePrefix + DIFF_PAGE + FILE_NAME_EXTENSION);
+        File result = new File(FILE_NAME_PREFIX + DIFF_PAGE + FILE_NAME_EXTENSION);
         ImageIO.write(diffImage, PNG_FORMAT, result);
         driver.quit();
 
+        compareHtmlPage(url1, url2);
+
         log.info("Screenshot for first page was saved: " + file1.getAbsolutePath());
         log.info("Screenshot for second page was saved: " + file2.getAbsolutePath());
         log.info("Differences result was saved: " + result.getAbsolutePath());
         log.info("Finished!");
         System.exit(0);
     }
 
+    private static String parseUrlToHtml(String url) throws IOException {
+        return Jsoup.connect(url).get().toString();
+    }
+
+    private static String[] convertHtmlToArray(String htmlAsString) {
+        return htmlAsString.split(BREAK_LINE);
+    }
+
+    public static void compareHtmlPage(String urlFirstPage, String urlSecondPage) throws IOException {
```

Comment:
Hi @tranquoccuonga3,
Could we extract this logic to the separated class so that we could reduce the class responsibility? (It's good practice to follow the Single Responsibility Principle)